scripts/generate-app-config/generatePagesConfig.js

```diff
@@ -136,6 +136,91 @@ const buildPageConfig = (filePaths, generatePagesOptions, namespace) => (
   }, {})
 );
 
+const getScreenshotPatterns = generatePagesOptions => (
+  generatePagesOptions.searchPatterns.reduce((acc, { root }) => {
+    const rootPath = root.replace(/[\\]/g, '/');
+    acc.push({
+      pattern: `${rootPath}/packages/*/tests/wdio/__snapshots__/reference/**/*.png`,
```

Comment:
This can be opened up now or wait till later?

Reply:
also... some projects use `test` instead of `tests` do we want to open this up or push people towards one?

@ryanthemanuel

Reply:
we should update the glob to account for tests and test

Reply:
And wait until someone complains that they really really need their test evidence folder in another location.